pkgs/by-name/mi/mieru/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "mieru";
+  version = "3.7.0";
+
+  src = fetchFromGitHub {
+    owner = "enfein";
+    repo = "mieru";
+    rev = "v${version}";
+    hash = "sha256-VFQWiihwd0YiLpgTxKUjVd+IkGivsa4jI/jb5ckjFt8=";
+  };
+
+  vendorHash = "sha256-u+faZ5KQQB2U8CdksZ4rqqPrOzxnlHOGHL7tAJobJ18=";
```

Comment:
hash mismatch

Reply:
> hash mismatch

I dont think so, since I rebuild on local and https://github.com/NixOS/nixpkgs/pull/295634#issuecomment-2444023294 shows matches well.

Reply:
Oh no